jest.config.js

```diff
@@ -175,10 +175,7 @@ module.exports = {
   // transform: undefined,
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  // transformIgnorePatterns: [
-  //   "/node_modules/",
-  //   "\\.pnp\\.[^\\/]+$"
-  // ],
+  transformIgnorePatterns: ['/node_modules/^(web-components)'],
```

Comment:
With the introduction of the React bindings, the import was breaking the Jest test. This ensures that the bindings get transformed through babel.

Reply:
:thinking: Ideally, the result of the web components React bindings build doesn't need to be transpiled in any way. We may need to fix this upstream.

Reply:
For more context, here is the problem that this fixes:

![image](https://user-images.githubusercontent.com/2008881/129791873-d4adf5b5-f7b7-4b79-90e9-0209dd165978.png)

I tried a few different things, like specifying a `transform` key for `babel-jest` but that didn't work.

I don't fully understand why Jest is complaining about an import in `node_modules/web-components/react-bindings/index.js` :shrug: